reflex/utils/format.py

```diff
@@ -253,24 +254,24 @@ def format_cond(
     # Use Python truthiness.
     cond = f"isTrue({cond})"
 
+    def create_cond_part_var(cond_part):
+        return Var.create_safe(
+            cond_part,
+            _var_is_string=type(cond_part) is str or isinstance(cond_part, Color),
```

Comment:
is this to ensure that if someone passes an instance of `Color` directly (instead of using the `color` function) that it still works?

Reply:
This is actually not needed since rx.color would be a Var by the time it gets here. Will push an update to take this out